### PR TITLE
Compact use of everywhere, delete stale calibrate scripts

### PR DESCRIPTION
**Pull request by ilopezgp**

### CODE REVIEW

**Review by charleskawczynski — APPROVED** [review]

**Comment by ilopezgp** [comment]
I am going to test the behavior a bit, and do it for the rest of the code.

**Comment by ilopezgp** [comment]
Nope, doesn't work unfortunately. I guess scoping issues as usual in Julia,

```
Precompiling project...
[32m  ✓ [39mCalibrateEDMF
  1 dependency successfully precompiled in 325 seconds (215 already precompiled)
ERROR: LoadError: On worker 2:
ArgumentError: Package ArgParse [c7e460c6-2fb9-53a9-8c5b-16f535851c63] is required but does not seem to be installed:
 - Run `Pkg.instantiate()` to install all recorded dependencies.

Stacktrace:
 [1] _require
   @ ./loading.jl:1089
 [2] require
   @ ./loading.jl:1013
 [3] #1
   @ /central/software/julia/1.7.0/share/julia/stdlib/v1.7/Distributed/src/Distributed.jl:79
 [4] #103
   @ /central/software/julia/1.7.0/share/julia/stdlib/v1.7/Distributed/src/process_messages.jl:274
 [5] run_work_thunk
   @ /central/software/julia/1.7.0/share/julia/stdlib/v1.7/Distributed/src/process_messages.jl:63
 [6] run_work_thunk
   @ /central/software/julia/1.7.0/share/julia/stdlib/v1.7/Distributed/src/process_messages.jl:72
 [7] #96
   @ ./task.jl:423

...and 9 more exceptions.

Stacktrace:
 [1] sync_end(c::Channel{Any})
   @ Base ./task.jl:381
 [2] macro expansion
   @ ./task.jl:400 [inlined]
 [3] _require_callback(mod::Base.PkgId)
   @ Distributed /central/software/julia/1.7.0/share/julia/stdlib/v1.7/Distributed/src/Distributed.jl:76
 [4] #invokelatest#2
   @ ./essentials.jl:716 [inlined]
 [5] invokelatest
   @ ./essentials.jl:714 [inlined]
 [6] require(uuidkey::Base.PkgId)
   @ Base ./loading.jl:1019
 [7] require(into::Module, mod::Symbol)
   @ Base ./loading.jl:997
 [8] top-level scope
   @ /central/software/julia/1.7.0/share/julia/stdlib/v1.7/Distributed/src/macros.jl:200
in expression starting at /central/home/ilopezgo/CalibrateEDMF.jl/experiments/scm_pycles_pipeline/julia_parallel/calibrate.jl:12
```

File: experiments/scm_pycles_pipeline/julia_parallel/calibrate.jl [hunk 1]
```diff
@@ -9,17 +9,19 @@
...
-@everywhere using ArgParse
-@everywhere using CalibrateEDMF
-@everywhere using CalibrateEDMF.DistributionUtils
-@everywhere using CalibrateEDMF.Pipeline
-@everywhere src_dir = dirname(pathof(CalibrateEDMF))
-@everywhere using CalibrateEDMF.HelperFuncs
-# Import EKP modules
-@everywhere using EnsembleKalmanProcesses
-@everywhere using EnsembleKalmanProcesses.ParameterDistributions
+@everywhere begin
```
Comment (charleskawczynski) [inline experiments/scm_pycles_pipeline/julia_parallel/calibrate.jl:12]: I think the Pkg and activate need to be done first, but the rest we should be able to batch

Comment (ilopezgp) [reply experiments/scm_pycles_pipeline/julia_parallel/calibrate.jl:12]: I will try that

Comment (ilopezgp) [reply experiments/scm_pycles_pipeline/julia_parallel/calibrate.jl:14]: same error

File: integration_tests/julia_parallel_calibrate.jl [hunk 2]
```diff
@@ -14,16 +14,18 @@ using HPC resources directly is advantageous.
...
-@everywhere using ArgParse
-@everywhere using CalibrateEDMF
-@everywhere using CalibrateEDMF.DistributionUtils
-@everywhere using CalibrateEDMF.Pipeline
-@everywhere src_dir = dirname(pathof(CalibrateEDMF))
-@everywhere using CalibrateEDMF.HelperFuncs
-# Import EKP modules
-@everywhere using EnsembleKalmanProcesses
-@everywhere using EnsembleKalmanProcesses.ParameterDistributions
+@everywhere Pkg.activate(dirname(dirname(dirname(@__DIR__))))
```
Comment (charleskawczynski) [inline integration_tests/julia_parallel_calibrate.jl:17]:
```suggestion
@everywhere Pkg.activate(dirname(@__DIR__))
```

Comment (charleskawczynski) [reply integration_tests/julia_parallel_calibrate.jl:17]: This may have actually been the issue before

**Comment by ilopezgp** [comment]
Ok, we are making progress! Now the experiment runs but we get this error in the plots,

```


ERROR: LoadError: ArgumentError: Package Plots not found in current path:
--
  | - Run `import Pkg; Pkg.add("Plots")` to install the Plots package.


```

**Pull request merged**